src/main/java/edu/usf/cutr/gtfs_realtime/bullrunner/GtfsRealtimeProviderImpl.java

```diff
@@ -452,6 +452,10 @@ private void refreshTripVehicle() throws IOException, JSONException {
 			 float bearing;
 			 for (int k = 0; k < vehicleArray.length(); k++) {
 					JSONObject vehicleObj = vehicleArray.getJSONObject(k);
+
+					// We only operate on "Route A" routes and ignore everything else

```

Comment:
Comment should technically be:

`// We only operate on "Route X" routes and ignore other strings`